### ISSUE TITLE
Add outline to selected results in details list on selection with mouse.

### ISSUE THREAD

**Issue by shrgup** [OfficeDev/office-ui-fabric-react#1804]
### Bug Report

- __Package version(s)__: (fill this out)
- __Browser and OS versions__: (fill this out if relevant)

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No)
NO
Requested priority: (Blocking, High, Normal, Low) 
BLOCKING
Products/sites affected: (if applicable)
VSTS
#### Describe the issue:

There is no outline or any way to differentiate the selected row in Details List from other rows except for color which is not enough to differentiate selected result in High contrast mode.

#### Actual behavior:

In HIgh Contrast mode , whenever we select a row using mouse , it is not clear if it selected or not .

#### Expected behavior:

There should be outline around the selected rows in High contrast mode.

**Comment by betrue-final-final** [comment]
Thank you, I agree and this is something we should fix. However, we don't want to use an outline as that will conflict with keyboard focus. Instead, we should use the cyan background and black text for High contrast black, and purple background for HC white, like Windows does. Here's a clip from File Explorer. Note keyboard focus and mouse hover, which do have outlines

![image](https://cloud.githubusercontent.com/assets/20363037/26158789/a593613e-3ad1-11e7-889b-97e8a0009e12.png)

**Comment by micahgodbolt** [comment]
@betrue-final-final are we using that cyan/purple color anywhere in fabric? Should we create variables for all of the colors in high contrast and black on white palettes?

**Comment by betrue-final-final** [comment]
The cyan and purple colors are useful for selection. I believe we are but I'd need to look up where. Note that we have 5 colors for HC blacks, and 8 for HC white we should have available.

![image](https://cloud.githubusercontent.com/assets/20363037/26324615/614d79ae-3ee8-11e7-82b0-41161698d863.png)

**Comment by shrgup** [comment]
Any ETA or progress around this?

**Comment by shrgup** [comment]
gentle reminder .. is this being tracked ?
Also , is it okay if we add outline to selected result as a temporary fix 
or any idea how involved is  the actual fix ?
Any pointers ?

**Comment by betrue-final-final** [comment]
Outline is focus, not selection. We really need to have the solid fill.
